### PR TITLE
Reenable any disabled darc-int Package Sources in SetupNugetSources

### PR DESCRIPTION
**Pull request by MattGal**
See https://github.com/dotnet/arcade/issues/5890 for context

### CODE REVIEW

File: eng/common/SetupNugetSources.sh [hunk 1]
```diff
@@ -146,3 +146,21 @@ for FeedName in ${PackageSources[@]} ; do
         sed -i.bak "s|$PackageSourceCredentialsNodeFooter|$NewCredential${NL}$PackageSourceCredentialsNodeFooter|" $ConfigFile
     fi
 done
+
+# Re-enable any entries in disabledPackageSources where the feed name contains darc-int
+# See https://github.com/dotnet/arcade/issues/5890 for context
+grep -i "<disabledPackageSources>" $ConfigFile
+if [ "$?" == "0" ]; then
+    DisabledDarcIntSources=()
+    echo "Re-enabling any disabled package sources in $ConfigFile"
```
Comment (riarenas) [inline eng/common/SetupNugetSources.sh:155]:
```suggestion
    echo "Re-enabling any "darc-int" disabled package sources in $ConfigFile"
```

File: eng/common/SetupNugetSources.ps1 [hunk 2]
```diff
@@ -94,6 +94,15 @@ function InsertMaestroPrivateFeedCredentials($Sources, $Creds, $Username, $Passw
     }
 }
 
+# See https://github.com/dotnet/arcade/issues/5890 for context
```
Comment (riarenas) [inline eng/common/SetupNugetSources.ps1:97]: [nit] I don't think a comment is really needed here, I'd rather add a blurb about this new functionality in the summary at the top of the script

File: eng/common/SetupNugetSources.sh [hunk 3]
```diff
@@ -146,3 +146,21 @@ for FeedName in ${PackageSources[@]} ; do
...
+    DisabledDarcIntSources+=$(grep -oh '"darc-int-[^"]*" value="true"' $ConfigFile  | tr -d '"')
+    for DisabledSourceName in ${DisabledDarcIntSources[@]} ; do
+        if [[ $DisabledSourceName == darc-int* ]]
+            then
+                OldDisableValue="add key=\"$DisabledSourceName\" value=\"true\""
+                NewDisableValue="add key=\"$DisabledSourceName\" value=\"false\""
+                sed -i.bak "s|$OldDisableValue|$NewDisableValue|" $ConfigFile
+                echo "Neutralized disablePackageSources entry for '$DisabledSourceName'"
```
Comment (riarenas) [inline eng/common/SetupNugetSources.sh:163]: What does neutralize mean here?

Comment (MattGal) [reply eng/common/SetupNugetSources.sh:163]: to counteract the activity or effect of : make ineffective

**Comment by MattGal** [comment]
Given team approval and no exercise of these two scripts in the PR workflow, I am merging.

**Pull request merged**